### PR TITLE
docs: replaced class hero to interface wherever class properties not used(#21186)

### PR DESCRIPTION
**Pull request by ajitsinghkaler**
…used(#21186)

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #21186

## Does this PR introduce a breaking change?

- [x] Yes Not Sure
- [ ] No

Replaced from class to interface if someplace using class properties then it may fail

### CODE REVIEW

File: aio/content/examples/dependency-injection/src/app/heroes/hero.ts [hunk 1]
```diff
@@ -2,5 +2,5 @@
 export class Hero {
   id: number;
   name: string;
-  isSecret = false;
+  isSecret:boolean = false;
```
Comment (ajitsinghkaler) [inline aio/content/examples/dependency-injection/src/app/heroes/hero.ts]: Can we remove this default and add it as an interface too

Comment (petebacondarwin) [reply aio/content/examples/dependency-injection/src/app/heroes/hero.ts]: Yes!

**Review by petebacondarwin — APPROVED** [review]

**Comment by petebacondarwin** [comment]
Could you also tweak the commit message. It should use present tense ("replace" rather than "replaced") and can you move the issue reference (#21186) into the body of the message and write:

```
Fixes #21186
```

This way the issue will be auto-closed when we land this PR.

**Comment by ajitsinghkaler** [comment]
@petebacondarwin I've done the changes Tell me if anything else needed

**Comment by petebacondarwin** [comment]
Merge assistance: minor docs only changes

**Pull request closed without merging**